### ISSUE TITLE
Package information in compilation error messages

### ISSUE THREAD

**Issue by caoilte** [arainko/ducktape#111]
Hi,
The compilation error messages are fantastic. We're spoilt compared to most macro errors just a few years ago. I have what is I hope a small suggestion for a tweak that I know would help me read the errors faster.

I've got a situation where I'm mapping to the same type name in multiple packages and at first glance I didn't spot that the compilation error message for `No instance of Transformer[X, Y] was found.` was about a type in a different package to the one I was working on. It would be really helpful if more package information was included in the compilation error messages.

There is actually some already. The first failure to find a Transformer is fully qualified. eg

```
[error]     |Neither an instance of Transformer[com.acme.wire.Event, com.acme.domain.Event] was found nor are 'Event' 'Event' 
[error]     |singletons with the same name.
```

However, I usually go straight to the last line of the error message as that is always the root cause. eg 

```
[error]     |Compiler supplied explanation (may or may not be helpful):
[error]     |No field named 'make' found in Vehicle
```

If `Vehicle` in this case was fully qualified (eg `com.acme.wire.Vehicle`) as well this would be really helpful i think.

**Comment by arainko** [comment]
Good point, I don't even remember why I made these not FQIs in the first place, I'll get a new release out over the weekend 😄

**Comment by caoilte** [comment]
I can totally see how it would be noisy in some situations as the types gets printed a lot in the error message. I assumed it was deliberate to reduce clutter and in general i like it.

**Comment by arainko** [comment]
This has been resolved in the `0.2.x` line (which just had its [RC1](https://github.com/arainko/ducktape/releases/tag/v0.2.0-RC1) release :smile: )
